portfolio/src/main/webapp/index.html

```diff
@@ -7,30 +7,31 @@
 		<script src="script.js"></script>
 	</head>
 	<body>
-	<div id="content">
-		<h1>Jenny's Portfolio</h1>
+		<div id="content">
+			<h1>Jenny's Portfolio</h1>
 
-		<table>
-			<tr>
-				<th onclick=>about me</th>
-			</tr>
-			<tr>
-				<th>contact me</th>
-			</tr>
-		</table>
+			<table>
+				<th onclick="ToggleReset()">home</th>
```

Comment:
can you parameterise these functions?

consider:
Toggle("about-page")
Toggle("contact-page")
Toggle(null)

The first thing toggle can do is hide everything, then if the arg is non-null, look up the arg and show it.